### ISSUE TITLE
performance tests FTBS on debian Testing

### ISSUE THREAD

**Issue by Trophime** [htool-ddm/htool#4]
For info, with g++ 9 on Debian Bullseye:
[htool.log](https://github.com/htool-ddm/htool/files/4677519/htool.log)

**Comment by PierreMarchand20** [comment]
Thank you for your feedback. Since performances tests cannot be built without asking for it specifically (there are not in the targets `all` or `build-tests` of cmake for example see https://github.com/htool-ddm/htool/blob/master/CMakeLists.txt#L275 ), I deduce you really want to use performance tests, right ?

- if it is the case, you are using the develop branch and I have not updated these tests yet  because they are not in the CI (it does not worth it to test performance on Travis I think). But I would not mind to add this to my todo list this week if needed. The version on master branch should work, but HPDDM changed its API so it may not work anymore with the last version of HPDDM.

- if it is not the case, then what command did you use exactly ? Because it should have not tried to build it by default if you wanted to use the examples and the functional tests (respectively `build-examples` and `build-tests`), and note that you do not need to compile anything if you just want to use the library since it is header-only. 

PS: I closed your previous issue because now I added testing on Debian, so I thought there would not be any issue now.
